lib/bamboozled/api/base.rb

```diff
@@ -78,6 +78,8 @@ def auth
       end
 
       def path_prefix
+        return @api_url unless @api_url.nil?
+
         "https://api.bamboohr.com/api/gateway.php/#{subdomain}/v1/"
```

Comment:
I wondering if it would be expected that `api_url` replaces the whole URL. I think it should just replace the first portion of it, so, the subdomain is not ignored. It could confuse someone who is using the gem.
wdyt?